### DISCUSSION TITLE
Allow silenced users to play Daily Challenges

### DISCUSSION THREAD

**Discussion by Userlog2** [ppy/osu#30627] [Ideas] [answered]
(Based on a discussion in the osu!dev Discord server around 14/11/2024 15 UTC)

If a user is silenced, they will not be able to play multiplayer matches.
Since playlists are similar to multiplayer, they cannot be played while silenced either.

Since Daily Challenges are an implementation of playlists, they cannot be played while silenced.
However, this not only causes the inability to play for the day, but also the breaking of a user's daily streak if they are silenced for 43 hours. This is not, and should not (in my opinion), be part of the punishment for less serious violations of the rules.
Therefore, I propose **the creation of a technical exception that allows silenced users to play Daily Challenges**.

## Responses to counter-arguments

This would not cause invalidation/compromising of the leaderboards, since restrictions are given for cheating or foul play; silences are not.

Regarding prevention of explicit/suggestive avatars: If a user is found to have an inappropriate avatar, their avatar will be reset along with their silence, meaning they will not be able to set any other inappropriate avatars for the length of the silence, if not longer (since moderators can place avatar restrictions that prevent changes to an avatar for longer periods of time). 
In addition, their avatar is visible everywhere else anyway (profile, comment, beatmap discussion, forum, **single-player rankings**, etc.) - why should they be hidden from just the Daily Challenge leaderboards?

**Comment by peppy** [comment, marked as answer]
Probably fine to allow. Without allowing chatting of course.

Can also include playlists in the allowance if that makes it easier.

cc @ppy/team-web
